rikka.proto

```diff
@@ -1,49 +1,49 @@
 syntax = "proto3";
 
 
-// This is a request for a question to be answered based off a passage 
-// currently passage max is 386 words and 50 for question 
+// This is a request for a question to be answered based off a passage
+// currently passage max is 386 words and 50 for question
```

Comment:
If this number is not consistent, I think it should be gotten rid of and instead just redirect people to the github settings file. Having an out-of-date comment is worse than having no comments.

Reply:
Address this in #5